sorbet/rbi/shims/server_test.rbi

```diff
@@ -0,0 +1,10 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Rails
+    class Server
+      def print_it!; end
```

Comment:
Open to better ways to handle this.

Reply:
But then we need to load sorbet-runtime, which I am aiming to avoid.

Reply:
Ah, right...well, then there's no other way other than using the shim or waiting until we have comment support for unsafe.